JavaShared/src/main/java/org/openzen/zenscript/javashared/prepare/JavaPrepareDefinitionVisitor.java

```diff
@@ -146,7 +146,7 @@ public class JavaPrepareDefinitionVisitor implements DefinitionVisitor<JavaClass
 			cls.addInstanceMethod("lastIndexOfStringFrom", "lastIndexOf", "(Ljava/lang/String;I)I");
 
 			cls.addInstanceMethod("replace", "replace", "(CC)Ljava/lang/String;");
-			cls.addInstanceMethod("trim", "trim", "()Ljava/lang/String;");
+			//cls.addInstanceMethod("trim", "trim", "()Ljava/lang/String;");
```

Comment:
trim was both a builtinMethodSymbol as well as added as `[Native]` in Strings.zs, causing ambiguous method calls due to the method existing twice.
Which one should we keep?